src/content.tsx

```
@@ -0,0 +1,100 @@
+        status: "added"
+      })
+    } catch (error) {
+      console.error(error)
+      setButtonState({
+        text: "Error to add the job into database",
+        status: "error"
+      })
+    }
+  }
+
+  useEffect(() => {
```

Biome

[error] This hook specifies more dependencies than necessary: window.location.href

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)